lms/static/scripts/frontend_apps/components/test/BasicLtiLaunchApp-test.js

```diff
@@ -176,56 +175,68 @@ describe('BasicLtiLaunchApp', () => {
     });
   });
 
-  it('reports the submission in the LMS when the content iframe starts loading', async () => {
-    fakeConfig.canvas.speedGrader.submissionParams = {
-      lis_result_sourcedid: 'modelstudent-assignment1',
-    };
+  describe('speed grader config', () => {
```

Comment:
This diff looks bad, but the only real change is bringing the speedGrader object down to only this code block